src/lighteval/tasks/lighteval_task.py

```diff
@@ -197,8 +198,19 @@ def __init__(  # noqa: C901
         self.metrics = as_list(cfg.metric)
         self.suite = as_list(cfg.suite)
         ignored = [metric for metric in self.metrics if Metrics[metric].value.category == MetricCategory.IGNORED]
+
         if len(ignored) > 0:
             hlog_warn(f"[WARNING] Not implemented yet: ignoring the metric {' ,'.join(ignored)} for task {self.name}.")
+
+        if any(
+            Metrics[metric].value.category in [MetricCategory.LLM_AS_JUDGE, MetricCategory.LLM_AS_JUDGE_MULTI_TURN]
```

Comment:
I really don't like that way of doing it but did not really find anything better. We would need a way to check for used metrics at loading and stop if the metric cannot be used.

Reply:
This works for me